README.md

```diff
@@ -1,81 +1,27 @@
-      G''==>J
-      E==>H{Attempt to<br/> save x509 instance cert<br/> to kubernetes secret};
-      H==>I[Success];
-      H-->I'[Failure];
-      I'-->C
-      H==>I''[No kubernetes secret configured];
-      I''==>J
-      I==>J{Attempt to<br/> retrieve x509 role certs<br/> from identity provider};
-      J==>K[Success];
-      J-->K'[Failure];
-      J==>K''[No roles configured];
-      K''==>N
-      K==>L{Write x509 instance/role certs to volume}
-      L==>M[Success];
-      L-->M'[Failure];
-      M'-->C
-      M==>C
-      K'==>N{Write x509 instance cert to volume}
-      N==>O[Success];
-      N-->O'[Failure];
-      O'-->C
-      O==>C
-```
+This repository contains [Athenz SIA](https://github.com/AthenZ/athenz/blob/master/docs/system_view.md#sia-service-identity-agent-provider) which is basically aimed to work on Kubernetes environment.
 
 ## Usage
+
+```
+$GOPATH/bin/athenz-sia --help
+```
+
+## Build
+
+### To update the git submodule from the latest commit (HEAD) hash of the remote repository
+
+```
+git submodule update --recursive --init --remote
+make
+```
+
+### To update the git submodule from the remote repository
+
 ```
-Usage of athenz-sia:
-  -backup-mode string
-    	Kubernetes secret backup mode, must be one of read or write (Note: Performing writes with a large number of concurrency may cause unexpected loads on k8s api) (default "read")
-  -delay-jitter-seconds int
-    	delay boot with random jitter within the specified seconds (0 to disable)
-  -delete-instance-id
-    	delete x509 cert record from identity provider when stop signal is sent (default true)
-  -dns-suffix string
-    	DNS Suffix for certs
-  -endpoint string
-    	Athenz ZTS endpoint
-  -log-dir string
-    	directory to store the server log files (default "/var/log/athenz-sia")
-  -log-level string
-    	logging level (default "INFO")
-  -mode string
-    	mode, must be one of init or refresh, required (default "init")
-  -out-ca-cert string
-    	CA cert file to write (default "/var/run/athenz/ca.cert.pem")
-  -out-cert string
-    	cert file to write (default "/var/run/athenz/service.cert.pem")
-  -out-cert-secret string
-    	Kubernetes secret name to backup cert (Backup will be disabled if empty)
-  -out-key string
-    	key file to write (default "/var/run/athenz/service.key.pem")
-  -out-rolecert-dir string
-    	directory to write cert file for role certificates (default "/var/run/athenz/")
-  -provider-service string
-    	Identity Provider service
-  -refresh-interval string
-    	cert refresh interval (default "24h")
-  -sa-token-file string
-    	bound sa jwt token file location (default "/var/run/secrets/kubernetes.io/bound-serviceaccount/token")
-  -server-ca-cert string
-    	path to CA cert file to verify ZTS server certs
-  -target-domain-roles string
-    	target Athenz roles with domain (e.g. athenz.subdomain:role.admin,sys.auth:role.providers)
+git submodule update --recursive --init
+make
```

Comment:
```suggestion
make submodule-update
```

Reply:
Rather than black-boxing the commands into the Makefile, it could be simpler to let developers execute the raw `git submodule` commands.